0.pre-preprocess_images/rename_files.sh

```diff
@@ -0,0 +1,16 @@
+#!/bin/bash
+
+echo "Renaming files..."
+
+jupyter nbconvert --to=script --FilesWriter.build_directory=scripts notebooks/*.ipynb
+
+dir=$(pwd)
+echo $dir
+cd scripts
+python change_file_names.py
+cd ../
+
+echo "Done!"
+
+# prep the output directory
```

Comment:
```suggestion
# prep the output directory for file type conversion in Fiji
```
Just a suggestion, current code comment is fine. This just add the extra explicit statement on what this next directory is for.